Makefile

```diff
@@ -95,7 +96,9 @@ podman: .gopathok API.md cmd/podman/varlink/ioprojectatomicpodman.go
 	$(GO) build -i -ldflags '$(LDFLAGS_PODMAN)' -tags "$(BUILDTAGS)" -o bin/$@ $(PROJECT)/cmd/podman
 
 python-podman:
+ifdef HAS_PYTHON3
 	$(MAKE) -C contrib/python python-podman
```

Comment:
Instead of making this a silent no-op, I'd rather drop `python-podman` from `binaries`, and require folks who want to build the binary python distribution file(s) to call `make python-podman` explicitly.  Callers who are looking for that output will notice it missing if they forget, and it saves a bit of compilation for everyone else.

Reply:
I considered that too.  But that in turn will require changes to fedora specs proper among other things.  Maybe I took a copout but this gets the job done.  What do others think?